.github/workflows/cron.yml

```diff
@@ -188,12 +189,19 @@ jobs:
           kind create cluster --config ./kind-config.yaml
       - name: Setup SSH
         run: |
+          
+          echo ">>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>"
           cat >>/tmp/setup.sh <<'END'
           NODES_IP="$(kubectl get nodes -A -o wide | awk 'FNR > 2 {print $6}')"
           NODES="$(kubectl get nodes -A -o wide | awk 'FNR > 2 {print $1}')"
+          
           for node in ${NODES}; do
+            echo ">>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>"
+            echo ">>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>"
             USER="$(whoami)"
-            docker cp /etc/apt/sources.list ${node}:/etc/apt/sources.list
+            docker exec ${node} cat /proc/version
+            docker exec ${node} rm -rf /usr/sbin/policy-rc.d
+            # docker cp scripts/ubuntu20sources.list ${node}:/etc/apt/sources.list
```

Comment:
remove unnecessary comments.

Reply:
This part will be fixed after we walk through the test workflow.